package.json

```
@@ -24,7 +24,7 @@
     "generate": "stencil generate"
   },
   "dependencies": {
-    "@stencil/core": "2.20.0",
+    "@stencil/core": "^2.20.0",
```

ini di buat static aja yah, takutnya ada major update dari stencil/core
```suggestion
    "@stencil/core": "2.20.0",
```